sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -135,6 +144,15 @@ export interface CertificateOperationError {
     readonly message?: string;
 }
 
+// @public
+export interface CertificateOperationPollOperationStatePublic extends PollOperationStateWithPublicState<CertificateOperationPublicState, KeyVaultCertificateWithPolicy> {
+}
+
+// @public
+export interface CertificateOperationPublicState {
+    operation?: CertificateOperation;
```

Comment:
since the operation is also the result shape in this case, should we call this `partialResult` or something?

Reply:
the operation is not the result type

Reply:
I don't like the word "Public" in the type name, if it were private it wouldn't be in the exported types. Could it just be CertificateOperationState? Or is the type really necessary? Why can't the CertificateOperation be the state be TPublic